frontend/packages/dev-console/src/components/topology/operators/actions/removeServiceBinding.tsx

```diff
@@ -1,35 +0,0 @@
-import * as React from 'react';
-import { confirmModal, errorModal } from '@console/internal/components/modals';
-import { YellowExclamationTriangleIcon } from '@console/shared/src';
-import { Edge } from '@patternfly/react-topology';
-import { k8sKill } from '@console/internal/module/k8s';
-import { ServiceBindingRequestModel } from '../../../../models';
-
-export const removeServiceBinding = (edge: Edge): Promise<any> => {
-  const message = (
-    <p>
-      Deleting the binding connector deletes the config details of the source and removes the
-      binding resources. Are you sure you want to delete the binding connector?
-    </p>
```

Comment:
Don't we need this custom message now for SBR while deletion?

Reply:
We use the standard delete now so, no this message is not needed.